components/FriendPagePreview.js

```diff
@@ -70,33 +88,62 @@ export default function FriendPagePreview({ friend }) {
           height={180}
         />
       </div>
-
       {/* <p>Contact:</p> */}
+
       <StyledContactList>
-        {Object.entries(friend.contactOptions).map(([key, value]) => (
-          <StyledList key={key}>
-            <StyledButton href={value}>
-              {key}{" "}
-              {/* : <br />
+        {friend.contactOptions &&
+          friend.contactOptions.map((options) => (
+            <StyledList key={options.name}>
+              <StyledButton href={options.number}>
+                {options.name}
+
+                {/* : <br />
               {value} */}
-            </StyledButton>
-          </StyledList>
-        ))}
+              </StyledButton>
+            </StyledList>
+          ))}
       </StyledContactList>
       <StyledInfoList>
-        <p>Birthday: {friend.birthday}</p>
-        <p>Status: {friend.friendship_status}</p>
-        <p>City: {friend.city}</p>
+        <p>
+          Birthday: <b>{friend.birthday}</b>
+        </p>
+        <p>
+          Status: <b>{friend.friendship_status}</b>
+        </p>
+        <p>
+          City: <b>{friend.city}</b>
+        </p>
         <p>Hobbies:</p>
         <ul>
-          {friend.hobbies.map((hobby, index) => (
-            <li key={index}>{hobby}</li>
-          ))}
+          {friend.hobbies &&
+            friend.hobbies.map((hobbie, index) => (
+              <li key={index}>
+                <b>{hobbie}</b>
+              </li>
+            ))}
         </ul>
       </StyledInfoList>
```

Comment:
As the name of your styled component `StyledInfoList` already suggests this could be a `ul` with `li` instead of `p` elements.